bin/ch/ch.cpp

```diff
@@ -757,6 +757,10 @@ HRESULT ExecuteTest(const char* fileName)
             IfFailGo(E_FAIL);
         }
 
+        if(HostConfigFlags::flags.TrackRejectedPromises)
+        {
+            ChakraRTInterface::JsSetHostPromiseRejectionTracker(WScriptJsrt::PromiseRejectionTrackerCallback);
+        }
         len = strlen(fullPath);
```

Comment:
nit: add a newline here

Reply:
Between the closing curly and the next line

Reply:
added